dataverse-webapp/src/main/webapp/createEditDataverse.xhtml

```diff
@@ -226,13 +215,13 @@
 
                         <div class="row">
                             <div class="col-sm-12 form-group">
-                                <label class="col-sm-3 control-label">#{bundle['dataverse.metadataElements']}</label>
-                                <div class="col-sm-9">
+                                <label class="col-md-3 control-label">#{bundle['dataverse.metadataElements']}</label>
+                                <div class="col-md-9">
                                     <p class="help-block">#{bundle['dataverse.metadataElements.tip']}</p>
 
                                     <label class="metadata-blocks-default" for="metadataRoot"
                                            jsf:rendered="#{CreateEditDataversePage.dataverse.owner != null}">
-                                        <h:selectBooleanCheckbox id="metadataRoot" tabindex="7"
+                                        <h:selectBooleanCheckbox id="metadataRoot" 
                                                                  styleClass="metadata-blocks-default"
                                                                  value="#{CreateEditDataversePage.mdbOptions.inheritMetaBlocksFromParent}"
                                                                  onclick="updateMetadata(this);">
```

Comment:
I can't see a checkbox for this field - only text is rendered "Use metadata fields from Root"

Reply:
Fixed.
These checkboxes were the ones I couldn't modify with pure CSS, I forgot to change them accordingly and that's why they were missing. I've now modified the code, so that even if any checkboxes would be overlooked they will appear in their default visual style, not vanish completely.